app/views/services/govbox/_hidden_fields_from_step1.html.erb

```diff
@@ -1,3 +1,4 @@
 <%= hidden_field_tag :mode, params[:mode], value: params[:mode] || 'sync_mode' %>
+<%= hidden_field_tag :referral_code, params[:referral_code], value: params[:referral_code] || cookies[:ref] %>
```

Comment:
Toto sa mi vobec nepaci, co je vlastne problem ze sa ta cookie nenastavi v prvom rade?

Reply:
Nevsimol som si, ze uz to je v podstate vyriesene len parameter musi mat format `ref` nie `referral_code`, cize toto uzavriem